### ISSUE TITLE
关于alpha的设置

### ISSUE THREAD

**Issue by wkiulu** [psyrocloud/MS-SSIM_L1_LOSS#1]
你好，请问损失函数中的alpha为什么要设置为0.025，我看《Loss Functions for Image Restoration With Neural Networks》这篇论文中好像设置为0.84，大了很多。
